File: src/main/java/com/devmpv/ChanReactor.java
package com.devmpv;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Application main class.
 * 
 * @author devmpv
 */
@SpringBootApplication
public class ChanReactor {

    protected ChanReactor() {
	super();
    }

    public static void main(final String[] args) {
	SpringApplication.run(ChanReactor.class, args);
    }

}

File: src/main/kotlin/com/devmpv/ChanReactor.kt
package com.devmpv

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * Application main class.
 *
 * @author devmpv
 */
@SpringBootApplication
class ChanReactor protected constructor() {
    companion object {

        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ChanReactor::class.java, *args)
        }
    }

}
